session/sync_upgrade.go

```diff
@@ -135,19 +134,17 @@ func isUpgradingClusterStateWithRetry(s sessionctx.Context, oldVer, newVer int64
 	for i := 0; ; i++ {
 		isUpgrading, err := IsUpgradingClusterState(s)
 		if err == nil {
-			if isUpgrading {
-				break
-			}
-			logger.Fatal("global state isn't upgrading, please send a request to start the upgrade first", zap.Error(err))
+			logger.Info("get global state", zap.Int64("old version", oldVer), zap.Int64("latest version", newVer), zap.Bool("is upgrading state", isUpgrading))
```

Comment:
if no err, will this log print too much?

Reply:
Just print the upgrade status and version information

Reply:
I mean too frequently.